code/modules/library/soapstone.dm

```diff
@@ -181,6 +151,8 @@
 	var/hash = md5(hidden_message)
 	var/newcolor = copytext(hash, 1, 7)
 	add_atom_colour("#[newcolor]", FIXED_COLOUR_PRIORITY)
+	light_color = "#[newcolor]"
+	set_light(1)
```

Comment:
Eh, I like the current amount of light they give out. You can only really notice it in the dark.